admin/pre_reg/views.py

```diff
@@ -160,11 +170,22 @@ def update_draft(request, draft_pk):
     else:
         schema_data = data.get('schema_data', {})
         data = deepcopy(draft.registration_metadata)
+        log_message = list()
         for key, value in data.items():
```

Comment:
This `for` seems to be losing comment data. Specifically, nested comments aren't retrieved.